scripts/jenkins/ardana/heat-ardana-dac.yaml

```diff
@@ -0,0 +1,259 @@
+heat_template_version: 2016-10-14
+
+description: >
+  Template for deploying Ardana in a multinode setup
+
+parameters:
+  key_name:
+    type: string
+    label: Key Name
+    description: Name of key-pair to be used for compute instance
+    default: engcloud-cloud-ci
+  image_id_controller:
+    type: string
+    label: Image ID
+    description: Image to be used for compute instance (controller)
+    default: cleanvm-jeos-SLE12SP3
+  image_id_compute:
+    type: string
+    label: Image ID
+    description: Image to be used for compute instance (compute node)
+    default: cleanvm-jeos-SLE12SP3
+  instance_type_controller:
+    type: string
+    label: Instance Type
+    description: Type of instance (flavor) to be used for the controller
+    default: cloud-ardana-job-controller
+  instance_type_compute:
+    type: string
+    label: Instance Type
+    description: Type of instance (flavor) to be used for compute nodes
+    default: cloud-ardana-job-compute
+  number_of_computes:
+    type: number
+    description: Count of compute nodes to create
+    default: 2
+  number_of_controllers:
+    type: number
+    description: Count of controller nodes to create
+    default: 1
```

Comment:
Not a HEAT expert, but can we inherit this section for some default and overwrite the number_of_* parameters?

Reply:
This is a good idea. Currently, the parameters are set in the JJB yaml file, which makes it hard to test new input models.
We should have a single input parameter - the input model name - and use that to import a <input-model>_vars.yml file created for each input model, that contains the number_of_... parameter valuse.

I'll see if I can do that in a separate PR.